src/browser/components/SavedScripts/SavedScripts.tsx

```diff
@@ -84,8 +84,11 @@ export default function SavedScripts({
   exportScripts,
   createNewFolder
 }: SavedScriptsProps): JSX.Element {
+  const folderExists = (folderId: string) =>
+    folders.find(folder => folder.id === folderId)
```

Comment:
Thanks for explaining the reason for the change, it makes sense to me. Minor nit would be that this find could be made more efficient using something like a map, but it's not worth blocking this PR for, especially since we plan to refactor this part of the code anyway :-)